src/components/Chip/index.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
+import type { MouseEventHandler } from 'react';
 
 import { Container, ChipContent } from './Chip.styles';
 
+interface Props {
+  children: React.ReactNode;
+  className: string;
+  isOverflowed: boolean;
```

Comment:
booleanType의 props를 사용자가 전달해야 할 때는 optional로 처리하는게 사용성에 더 좋을 것 같슴다!

```suggestion
  isOverflowed?: boolean;
```

Reply:
@kimjeongwonn 옵셔널 처리하는 부분들을 생각을 못했네요 ㅠㅠ 감사합니다